src/cmp.rs

```diff
@@ -19,22 +19,22 @@ use std::collections::HashMap;
 
 #[derive(Debug, PartialEq)]
 pub struct CompareResult {
-    /// Names of icons present in the first font but not the second.
+    /// Names of icons present in rhs but not lhs font.
```

Comment:
Highly recommend use of before/after or new/old or something of that nature instead of lhs/rhs. I know what you mean but ... it's a bit obtuse.